src/main/java/com/learning/mfscreener/service/PortfolioService.java

```diff
@@ -36,50 +33,61 @@
 import org.springframework.web.multipart.MultipartFile;
 
 @Service
+@Loggable
 public class PortfolioService {
```

Comment:
@coderabbitai, please suggest descriptive variablenames for all methods in this class.